python/shotgun_fields/image_widget.py

```diff
@@ -215,11 +245,17 @@ def _display_value(self, value):
 
         :param value: The value returned by the Shotgun API to be displayed
         """
-        # queue up the download in the background
-        entity_id = self._entity and self._entity.get("id") or None
-        entity_type = self._entity and self._entity.get("type") or None
-        self._task_uid = self._data_retriever.request_thumbnail(
-            value, entity_type, entity_id, self._field_name, load_image=True)
+
+        if isinstance(value, QtGui.QPixmap):
```

Comment:
this value seems unexpected given the docstring - maybe docstring needs updating?